### DISCUSSION TITLE
How to interpret the frequency domain dump

### DISCUSSION THREAD

**Discussion by linhz0hz** [thliebig/openEMS-Project#78] [Q&A Usage]
I am trying to simulate a log periodic antenna, and I though it would be helpful to see where the field is concentrated at for a given frequency. So I added a frequency-domain dump on top the time-domain dump

```
f_dump = np.linspace(max(1e9,f0-fc),f0+fc,101)
Ef = CSX.AddDump('Ef', file_type=0, dump_type=10,frequency = f_dump,sub_sampling=[2,2,2])
Ef.AddBox(start, stop);
```

After running the sim, I got a huge amount of files. It seems for each frequency, I got a file with suffix _avg, _abs, and _p=0 to _p=300 something. What does the p value stand for? And how does it work internally? 
Is it simply a fourier transform of the time domain data? If that is the case, I would expect a real and a imaginary component. 

I tried to search for answers but I could not. It would be great if someone can explain this.

**Comment by thliebig** [comment]
Hi, looking at fields is always a good idea. The only question is if it has to be 101 frequencies or if one or two would be enough?
Processing all this data and save them in memory is a huge slowdown...
If you need this many frequencies than file_type 0 (vtk) is not ideal either, maybe better use the hdf5 format (file type 1) as it is binary and saves all the data in one file (real and imag for every freq).
The vtk file is really meant to quickly load them in Paraview and see the animated fields. The suffix _abs is clear I think. p=0..360 is the phase looped over multiple angles. If you load these in Paraview you get the animation directly...

I hope that helps?
br
Thorsten
